.github/workflows/fcos-podman-next-build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build FCOS image with packages from rhcontainerbot/podman-next
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_NAME: fcos
+  IMAGE_TAGS: podman-next ${{ github.sha }}
+  IMAGE_REGISTRY: quay.io/podman
+  REGISTRY_USER: ${{ secrets.QUAY_PODMAN_USERNAME }}
+  REGISTRY_PASSWORD: ${{ secrets.QUAY.PODMAN_PASSWORD }}
+  COPR_OWNER: rhcontainerbot
+  COPR_PROJECT: podman-next
+
+jobs:
+  wait-for-task:
```

Comment:
Minor: The job-name is a little confusing, since it's doing all the steps.

Reply:
changing in update.